src/Calendar.jsx

```diff
@@ -390,7 +388,7 @@ let Calendar = React.createClass({
     let ToolbarToRender = components.toolbar || Toolbar
 
     return (
-      <div {...elementProps}
```

Comment:
Ohh so you mean something like this?

```
// Calendar.jsx

let Calendar = React.createClass({

  propTypes: {
    elementProps: PropTypes.object,
    ...
  }

  render() {
    let { className, style, elementProps, ...props } = this.props;
    ...
    return (
      <div
        {...elementProps}
        className={cn('rbc-calendar', className, {
          'rbc-rtl': props.rtl
        })}
        style={style}
      >
        ...
      </div>
    );
  }

});
```

And then user passes in `elementProps` in the component

```
// UserCalendar.jsx

import ReactBigCalendar from 'react-big-calendar'

let UserCalendar = React.createClass({

  render() {
    let elementProps = {
      onDrop: function() { ... },
      'data-id': 123
    };
  
    return (
      <ReactBigCalendar elementProps={elementProps} ... />
    );
  }
  
});
```